nvdaHelper/vbufBase/backend.cpp

```diff
@@ -26,7 +26,7 @@ using namespace std;
 
 VBufBackendSet_t VBufBackend_t::runningBackends;
 
-VBufBackend_t::VBufBackend_t(int docHandleArg, int IDArg): renderThreadID(GetWindowThreadProcessId((HWND)UlongToHandle(docHandleArg),NULL)), rootDocHandle(docHandleArg), rootID(IDArg), lock(), renderThreadTimerID(0), invalidSubtreeList() {
+VBufBackend_t::VBufBackend_t(int docHandleArg, int IDArg): renderThreadID(GetWindowThreadProcessId((HWND)UlongToHandle(docHandleArg),NULL)), rootDocHandle(docHandleArg), rootID(IDArg), lock(), renderThreadTimerID(0), pendingInvalidSubtreesList() {

```

Comment:
nit: We initialise pendingInvalidSubtreesList here, but not workingInvalidSubtreesList. It'd be good to be consistent here. I don't think they need to be initialised because they're C++ types and thus have a defined initial state.